src/components/Requirements/MultiplePlansDropdown.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div>
+    <div class="multiplePlans-dropdown">
+      <div
+        class="multiplePlans-dropdown-placeholder wrapper"
+        @click="closeDropdownIfOpen()"
+        data-cyid="multiplePlans-wrapper"
+        :class="{ plansHidden: shown }"
+      >
+        <img :src="editplan" class="multiplePlans-dropdown-placeholder editimg" />
+        <div class="multiplePlans-dropdown-placeholder plan">{{ currplan }}</div>
+        <div class="multiplePlans-dropdown-placeholder down-arrow" v-if="!shown"></div>
+        <div class="multiplePlans-dropdown-placeholder up-arrow" v-if="shown"></div>
+      </div>
+      <div class="multiplePlans-dropdown-content" v-if="shown">
+        <div
+          v-for="plan in filterPlans"
+          class="multiplePlans-dropdown-content item"
+          :key="plan"
+          @click="planClicked(plan)"
+        >
+          {{ plan }}
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+import editplan from '@/assets/images/editplan.svg';
+
+export default defineComponent({
+  props: {
+    plan: { type: String, default: 'No additional plans yet' },
+  },
+  data() {
+    return {
+      plans: ['PLAN 1', 'PLAN 2'],
+      shown: false,
+      currplan: 'PLAN 1',
+      editplan,
+    };
+  },
+  computed: {
+    filterPlans() {
```

Comment:
Can you rename this to something more descriptive like `otherPlans` or `deselectedPlans`?